md/README.md

```diff
@@ -59,6 +59,8 @@ The mode parameter has four options:
 
 The initialization of `MentionAnnotator()` without parameter set mode to `"ACE_NONTYPE"`
 
+There is a static function `getHeadConstituent(Constituent extent, String viewName)` built into the annotator. It can be used to get a constituent that is only the head, with all the attributes that the extent constituent has.
```

Comment:
"get a constituent that is only the head" -> get the head constituent

Reply:
What do you mean by "with all the attributes that the extent constituent has"?

Reply:
Ah I mean attributes like "MentionType" is copied to the new head constituent

Reply:
I think you can drop that and just say: 
"... built into the annotator which returns the mention-heads. "